portfolio/src/main/webapp/index.html

```diff
@@ -9,13 +9,27 @@
   <body>
     <div id="content">
       <h1>Ryan's Portfolio</h1>
-      <p>Hello! My name is Ryan Simone and I am currently a Google Step Intern. I am a rising junior at the University of California Santa Barbara pursuing a Bachelor's degree in Computer Science.</p>
-      <p>I was born in Torrance, CA on August 18, 1999. I am the youngest, yet tallest, member of my family. As a child, I was nicknamed "Ry-Ry the beach going guy" because of my love for the beach. I played competetive soccer until college. </p>
-      <p>To visit my github page, click 
+      <p>Hello! My name is Ryan Simone and I am currently a Google Step Intern.</p>
+      <p>I am a rising junior at the University of California Santa Barbara pursuing a Bachelor's degree in Computer Science.</p>
+      <p>I'm the youngest, yet tallest, member of my family. I've lived in Southern California my entire life.</p>
+      <h2>Hobbies</h2>
+      <dl>
```

Comment:
From what I read on [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl), the `<dl>` element (which I don't think I've heard of before) seems to be geared toward situations where you have a series of items, at least some of which have some sort of description attached. I think it would be good to split out some of the items to have details, e.g. `<dt>I love watching sports.</dt><dd>My favorite is soccer.<dd>`, or change this to a `<ul>` with `<li>`s for the items.

Reply:
Yea I agree the unordered list tag is better for this.